images/varnish/docker-entrypoint

```diff
@@ -2,4 +2,4 @@
 
 ep /etc/varnish/*
 
-exec "$@"
+exec /usr/sbin/varnishd -a :8080 -T :6082 -F -f /etc/varnish/default.vcl -S /etc/varnish/secret -p http_resp_hdr_len=$HTTP_RESP_HDR_LEN -p http_resp_size=$HTTP_RESP_SIZE
```

Comment:
we can't have that inside an entrypoint, or we will never be able to start a container with `docker run --rm -it lagoon/varnish sh`

so we need to create it's own shell script that is then used in `CMD`

Reply:
updated and fixed a few smaller things on the way there